crates/tests-e2e/tests/zzz_stop_orders.rs

```diff
@@ -0,0 +1,49 @@
+// XXX: This file starts with `zzz` as the tests are run in alphabetical order.
+// This test kills and starts the maker process again, and we do not want to interfere with other
+// tests.
+
+use native::api::ContractSymbol;
+use orderbook_commons::Prices;
+use tests_e2e::process::is_maker_running;
+use tests_e2e::process::kill_process;
+use tests_e2e::setup::TestSetup;
+use tests_e2e::wait_until;
+
+fn has_any_price(prices: &Option<Prices>) -> bool {
+    // Hardcoded to BTCUSD
+    if let Some(prices) = prices {
+        assert!(prices.len() == 1, "only BTCUSD is supported for now");
+        let btcusd = prices
+            .get(&ContractSymbol::BtcUsd)
+            .expect("BTCUSD price to be present");
+        return btcusd.bid.is_some() || btcusd.ask.is_some();
+    }
+    false
+}
+
+#[tokio::test]
+#[ignore = "need to be run with 'just e2e' command"]
+async fn no_orders_when_counterparty_is_offline() {
+    let setup = TestSetup::new_after_funding().await;
+    wait_until!(has_any_price(&setup.app.rx.prices()));
+
+    tracing::info!("Killing maker process, so that it does not send any orders");
+    kill_process("maker").expect("to be able to kill maker process");
+    assert!(!is_maker_running(), "maker should be stopped by now");
+
+    let wait_time = std::time::Duration::from_secs(70);
```

Comment:
❓ Would it be possible to make the expiry time configurable so we can set it to e.g. `5` seconds instead so this test does not have to wait for `70` seconds? 😅

Reply:
it would be nice indeed, I will create a ticket for this!

Reply:
I'm against having a test which runs for 70+ seconds :/

Reply:
Moved the test into a separate PR #937 

I also realised that I don't need to wait for that long, because maker is likely to have already sent an order.